internal/backend/windows/wslexe_windows.go

```diff
@@ -45,3 +50,56 @@ func (Backend) SetAsDefault(distroName string) error {
 	}
 	return nil
 }
+
+// State returns the state of a particular distro as seen in `wsl.exe -l -v`.
+func (Backend) State(distributionName string) (s state.State, err error) {
+	cmd := exec.Command("wsl.exe", "--list", "--all", "--verbose")
+	cmd.Env = append(cmd.Env, "WSL_UTF8=1")
+
+	out, err := cmd.Output()
+	if err != nil {
+		return s, err
+	}
+
+	/*
+		Sample output:
+		   NAME           STATE           VERSION
+		 * Ubuntu         Stopped         2
+		   Ubuntu-Preview Running         2
+	*/
+
+	sc := bufio.NewScanner(bytes.NewReader(out))
+
+	// Discarding header
+	sc.Scan()
```

Comment:
rather than this, I think it will be clearer to have a `var headerSkipped bool` and in the loop:

```go
// Discarding header
if !headerSkipped {
    headerSkipped = true
    continue
}
```

That way, we only scan in the for loop.

Reply:
Sounds good.